game.js

```diff
@@ -56,6 +39,11 @@ Player2.prototype.placeMarker = function(){
 		console.log("invalid input");
 		selectPosition();
 	}
+};
```

Comment:
This function's function (haha) would ask the player where he wants to place his marker - potentially validate that the position isn't already taken by checking the board and asking again if it is, then it should "return" the position the player chose.